### PR TITLE
softscope

### PR DESCRIPTION
**Pull request by pfitzseb**
Fixes https://github.com/JunoLab/Juno.jl/issues/174.

### CODE REVIEW

**Comment by stevengj** [comment]
Thanks, this seems like a good step.

(Alternatively, one could apply everything transformation function in the `REPL.repl_ast_transforms` list, which by default is just `REPL.softscope`.  That way, users could modify the transformations by changing this list.  See also JuliaLang/IJulia.jl#947)

File: src/repl.jl [hunk 1]
```diff
@@ -220,7 +220,7 @@ function evalrepl(mod, line)
       end
       errored && return nothing
       try
-        ans = repleval(mod, line)
+        ans = repleval(mod, REPL.softscope(line))
```
[the remark this answers is not in the document]
Comment (stevengj) [reply src/repl.jl:223]: It's not defined for Julia < 1.5.  (Maybe you meant ≤)

[the remark this answers is not in the document]
Comment (aviatesk) [reply src/repl.jl:223]: yes

[the remark this answers is not in the document]
Comment (pfitzseb) [reply src/repl.jl]: Done.

**Review by aviatesk — APPROVED** [review]

**Pull request merged**